holopy/scattering/imageformation.py

```diff
@@ -154,15 +154,15 @@ def _pack_scattering_matrix_into_xarray(
             self, scat_matrs, r_theta_phi, schema):
         flattened_schema = flat(schema)
         point_or_flat = self._is_detector_view_point_or_flat(flattened_schema)
-        dims = [point_or_flat, 'Epar', 'Eperp']
+        dims = [point_or_flat, 'E_out', 'E_in']
 
         coords = {point_or_flat: flattened_schema.coords[point_or_flat]}
         coords.update({
             'r': (point_or_flat, r_theta_phi[0]),
             'theta': (point_or_flat, r_theta_phi[1]),
             'phi': (point_or_flat, r_theta_phi[2]),
-            'Epar': ['S2', 'S3'],
-            'Eperp': ['S4', 'S1'],
+            'E_out': ['parallel', 'perpendicular'],
```

Comment:
I much prefer this new naming convention, especially because (i) it is clear and (ii) these are the actual dimensions (e.g. in=parallel, out=perpendicular is correct and readable, not in=S1, out=S3 (say) which is both incorrect and not readable).

But I'm not sure if others want the S1, S2, S3, S4 etc convention explicitly stated somewhere, see the discussion in #380.

Reply:
I put a comment in the code. I could also refer to GitHub issue #380 since there's a nice diagram there that makes everything clear. What do you think?

Reply:
I think this is fine.